src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,27 +31,13 @@
  */
 public class RobotContainer {
     // Subsystems
-    //private final SwerveSubsystem swerveSubsystem;
-    private final MissileShellSwerveSubsystem swerveSubsystem;
+    private final Tank tank = new Tank();
+    private final Intake intake = new Intake();
+    private final Carriage carriage = new Carriage();
 
     // Controllers and buttons
-    private final XboxController driveController = new XboxController(0);
-    private final JoystickButton 
-        driveAButton = new JoystickButton(driveController, XboxController.Button.kA.value),
-        driveBButton = new JoystickButton(driveController, XboxController.Button.kB.value),
-        driveXButton = new JoystickButton(driveController, XboxController.Button.kX.value),
-        driveYButton = new JoystickButton(driveController, XboxController.Button.kY.value),
-        driveLBumper = new JoystickButton(driveController, XboxController.Button.kLeftBumper.value),
-        driveRBumper = new JoystickButton(driveController, XboxController.Button.kRightBumper.value);
-
-    private final XboxController mechController = new XboxController(1);
-    private final JoystickButton 
-        mechAButton = new JoystickButton(mechController, XboxController.Button.kA.value),
-        mechBButton = new JoystickButton(mechController, XboxController.Button.kB.value),
-        mechXButton = new JoystickButton(mechController, XboxController.Button.kX.value),
-        mechYButton = new JoystickButton(mechController, XboxController.Button.kY.value),
-        mechLBumper = new JoystickButton(mechController, XboxController.Button.kLeftBumper.value),
-        mechRBumper = new JoystickButton(mechController, XboxController.Button.kRightBumper.value);
+    public final XboxController controller = new XboxController(0);
```

Comment:
You'll end up having two controllers anyways (mech and drive), so technically this boilerplate that constructs both controllers and all the buttons didn't have to be removed.